modules/monitoring-configuring-remote-write.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="configuring_remote_write_storage_{context}"]
+= Configuring remote write storage
+
+You can configure remote write to enable Prometheus to send ingested metrics to remote systems for storage. 
+Doing so has no impact on how or for how long Prometheus stores metrics using local storage.
+
+.Prerequisites
+
+* *If you are configuring core OKD monitoring components:*
+** You have access to the cluster as a user with the `cluster-admin` role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
+* *If you are configuring components that monitor user-defined projects:*
+** You have access to the cluster as a user with the `cluster-admin` role or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
+** You have created the `user-workload-monitoring-config` `ConfigMap` object.
+* You have installed the OpenShift CLI (`oc`).
+* You have set up a remote write compatible endpoint (such as Thanos) and know the endpoint URL.
```

Comment:
What are valid 'remote write compatible endpoints'? Can we define or outline the requirements of what is acceptable here better?

Reply:
What specification or RFC is our API providing?

Reply:
@jan--f - can you provide more detail to address these questions?

Reply:
The actual specification is still somewhat being worked out. See https://docs.google.com/document/d/1LPhVRSFkGNSuU1fBd81ulhsCPR4hkSZyyBj1SZ8fWOM/edit#heading=h.3p42p5s8n0ui
Its probably not too useful to users to have a link to the protocol specification, at least not at this point.

What is the goal of putting this into the user facing docs? Maybe we can find an alternative solution, say linking to the l[ist of `remote_write` compatible endpoints](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage)?

Reply:
Agree with @jan--f . Added the suggested to link to the remote_write compatible endpoints.